### ISSUE TITLE
writeq/1 trouble

### ISSUE THREAD

**Issue by flexoron** [trealla-prolog/trealla#620]
v1.34.11
```
?- writeq('\1\').
'\x1\'   true.

?- writeq('\0\').
'\x0\'   true.

?- op(1,xfx,'').
   true.

?- writeq('\0\').
''   true.   % How come?

?- writeq('\1\').
'\x1\'   true.
```

**Comment by flexoron** [comment]
v1.34.13
```
?- op(1,xfx,'').
   true.

?- char_code(Ch,0).
   Ch = '\x0\'.  % round brackets blown away!

?- writeq(' \x0\').
' \x0\'   true.

?- writeq(' \x0\ ').
' \x0\ '   true.

?- writeq('\x0\').
''   true. % unexpected?

?- writeq('\x0\ ').
''   true. % unexpected?
?- 
```

**Comment by flexoron** [comment]
```
GNU (1.5.1)

| ?- op(1,xfx,'').
| ?- writeq('''' '' '''').
'''' '' ''''

Trealla (v1.34.17)

?- op(1,xfx,'').
?- writeq('''' '' '''').
'\'' '''\''   true.
```

**Comment by UWN** [comment]
Same issue as [#132](http://www.complang.tuwien.ac.at/ulrich/iso-prolog/conformity_testing#132).

```
?- op(100,fx,' op'). 
   true.
?- /**/ writeq(' op' '1'). 
' op''1'   true. % unexpected
```
It is unclear what the goal of Trealla is.

**Comment by flexoron** [comment]
Fixed with v1.34.19
